api/routes/userRouter.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const { userController } = require("../controllers");
+
+const router = express.Router();
+
+router.post("/signup", userController.kakaoSignUp);
```

Comment:
```suggestion
router.post("/kakao/signin", userController.kakaoSignIn);
```
1. 플랫폼이 많아지는 경우를 고려해서, 애초에 `/kakao`라고 분기시켜주시는게 좋습니다. 추후에 여러 플랫폼이 생긴다면 백엔드 코드만 `/signin`에서 `/kakao/signin`으로 수정되면 끝나는게 아닌, 프론트엔드 소스코드에서 fetch하는 부분도 수정이 있어야한다는 점도 고려해야합니다.
2. 보통 OAuth를 통한 소셜 로그인이라고 한다면 하나의 엔드포인트에서 회원 가입과 로그인처리까지 이루어질텐데 빈도를 생각한다면 signin이라고 작성되는게 나을 것 같긴합니다.